src/graph/on_disk_graph.cpp

```diff
@@ -30,11 +31,15 @@ namespace graph {
 static std::unique_ptr<RelTableScanState> getRelScanState(MemoryManager& mm,
     const TableCatalogEntry& relEntry, const RelTable& table, RelDataDirection direction,
     ValueVector* srcVector, ValueVector* dstVector, ValueVector* relIDVector,
-    expression_vector properties, const Schema& schema, const ResultSet& resultSet) {
+    expression_vector predicateEdgeProperties, std::optional<column_id_t> edgePropertyID,
+    ValueVector* propertyVector, const Schema& schema, const ResultSet& resultSet) {
     auto columnIDs = std::vector<column_id_t>{NBR_ID_COLUMN_ID, REL_ID_COLUMN_ID};
-    for (auto property : properties) {
+    for (auto property : predicateEdgeProperties) {
         columnIDs.push_back(property->constCast<PropertyExpression>().getColumnID(relEntry));
     }
+    if (edgePropertyID) {
```

Comment:
@benjaminwinger I would call them `edgePropertyColumnID`. `PropertyID` is terminology in the binding phase. In the processor we usually use `ColumnID`. 

If u decide to change, apply this change across this PR.